### ISSUE TITLE
Data from get_all_results_from_query_batch does not contain newline/carriage return 

### ISSUE THREAD

**Issue by bakobako** [heroku/salesforce-bulk#108]
Hi I have a text data field in a Salesforce object that contains newlines, e.g.

"this is line1
this is line 2
this is line 3"

But when I create a query to return the data of the object that specific textfield does not contain the newlines any more and the data is squashed to this : 
"this is line1this is line 2this is line 3"

Is there any way I can overcome this and fetch the newlines? Is this a bug or the way the API itself works

**Comment by lambacck** [comment]
I don't think this library is doing anything that would remove the new lines and it's being used elsewhere in cases where the newlines are being preserved. Can you provide more details about what you are doing? Perhaps some sample code?
